api/task_lifecycle_client.go

```diff
@@ -59,29 +59,12 @@ func (c *TaskLifecycleClient) Scheme() string {
 	return c.url.Scheme
 }
 
-// CreateTaskSimple takes a task request and run option and sends this information to the client. It then returns
+// CreateTask takes a task request and run option and sends this information to the client. It then returns
 // a task response object and any errors to the caller.
 // TODO: remove this to conform to interface
 func (c *TaskLifecycleClient) CreateTask(ctx context.Context, runOption string, req TaskRequest) (*oapigen.CreateTaskResponse, error) {
-	var run oapigen.CreateTaskParamsRun
-	switch runOption {
-	case RunOptionInspect:
-		run = RunOptionInspect
-	case RunOptionNow:
-		run = RunOptionNow
-	case "":
-		run = ""
-	default:
-		err := errors.New("invalid run option provided")
-		return nil, err
-	}
-
-	resp, err := c.CreateTaskWithResponse(ctx, &oapigen.CreateTaskParams{Run: &run}, oapigen.CreateTaskJSONRequestBody(req))
-	if err != nil {
-		return nil, err
-	}
-
-	return resp, nil
+	run := oapigen.CreateTaskParamsRun(runOption)
```

Comment:
Swagger validation happens as part of `c.CreateTaskWithResponse()` and that validates the run parameter is in the enum of supported options, no need for our additional validation logic.